lms/templates/help_modal.html

```diff
@@ -18,197 +18,234 @@
+          )}</p>
+        <hr>
 
-    <p>${Text(_('Have a {strong_start}question about something specific{strong_end}? You can contact the {platform_name} general support team directly:')).format(
-        strong_start=HTML('<strong>'),
-        strong_end=HTML('</strong>'),
-        platform_name=static.get_platform_name()
-      )}</p>
-    <hr>
+        <div class="help-buttons">
+          <a href="#" id="feedback_link_problem">${_('Report a problem')}</a>
+          <a href="#" id="feedback_link_suggestion">${_('Make a suggestion')}</a>
+          <a href="#" id="feedback_link_question">${_('Ask a question')}</a>
+        </div>
 
-    <div class="help-buttons">
-      <a href="#" id="feedback_link_problem">${_('Report a problem')}</a>
-      <a href="#" id="feedback_link_suggestion">${_('Make a suggestion')}</a>
-      <a href="#" id="feedback_link_question">${_('Ask a question')}</a>
+        <p class="note">${_('Please note: The {platform_name} support team is English speaking. While we will do our best to address your inquiry in any language, our responses will be in English.').format(
+            platform_name=static.get_platform_name()
+          )}</p>
     </div>
 
-    <p class="note">${_('Please note: The {platform_name} support team is English speaking. While we will do our best to address your inquiry in any language, our responses will be in English.').format(
-        platform_name=static.get_platform_name()
-      )}</p>
+    <div id="feedback_form_wrapper">
 
-  </div>
+        <header></header>
 
-  <div class="inner-wrapper" id="feedback_form_wrapper">
-    <button class="close-modal">
-      <span class="icon fa fa-remove" aria-hidden="true"></span>
-      <span class="sr">
-        ## Translators: this is a control to allow users to exit out of this modal interface (a menu or piece of UI that takes the full focus of the screen)
-        ${_('Close')}
-      </span>
-    </button>
+        <form id="feedback_form" class="feedback_form" method="post" data-remote="true" action="/submit_feedback">
+          <div class="feedback-form-error" aria-live="polite">
+              <div id="feedback_error" class="modal-form-error" tabindex="-1"></div>
+          </div>
+    % if not user.is_authenticated():
+          <label data-field="name" for="feedback_form_name">${_('Name')}*</label>
+          <input name="name" type="text" id="feedback_form_name" required>
+          <label data-field="email" for="feedback_form_email">${_('E-mail')}*</label>
+          <input name="email" type="text" id="feedback_form_email" required>
+    % endif
+          <label data-field="subject" for="feedback_form_subject">${_('Briefly describe your issue')}*</label>
+          <input name="subject" type="text" id="feedback_form_subject" required>
+          <label data-field="details" for="feedback_form_details">${_('Tell us the details')}*
+    <span class="tip">${_('Include error messages, steps which lead to the issue, etc')}</span></label>
```

Comment:
@waheedahmed 
How about: "Describe what you were doing when you encountered the issue. Include any details that will help us to troubleshoot, including error messages that you saw."